### PR TITLE
Admin panel option to hide / show panel headings by Operating System

### PR DESCRIPTION
**Pull request by jacques42**
#### Prerequisites checklist

- [X] I have read the [contributing guidelines](https://github.com/andi34/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)

- [ ] Documentation update
- [ ] Bug fix
- [X] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)
For continued platform compatibility I thought it would be worthwhile to add an admin panel feature that does allow to show or hide admin panel headings (feature settings) by platform / operating system. 

This PR would add a new parameter to `configsetup.inc.php` for each panel-heading, to indicate platform compatibility:
```
'platform' => 'all'
```
Accepted values are `'all'`, `'linux'`, `'windows'`. Example:
```
        'remotebuzzer' => [
                'platform' => 'linux',
                'remotebuzzer_enabled' => [
                        'type' => 'checkbox',
                        'name' => 'remotebuzzer_enabled',
                        'value' => $config['remotebuzzer_enabled']
                ],
```

A missing `platform` key in the panel heading section defaults to `all` as value

Depending on a platform match, the panel heading / section is shown on the admin page or not.

#### Is there anything you'd like reviewers to focus on?
I don't think there is but a double-check to ensure I did not create some incompatibilities elsewhere would be great.

### CODE REVIEW

File: admin/index.php [hunk 1]
```diff
@@ -64,6 +64,10 @@
 				<?php
 					$i = 0;
 					foreach($configsetup as $panel => $fields) {
+						if (! empty($fields['platform']) && $fields['platform'] != 'all' && $fields['platform'] != $os) {
```
Comment (andi34) [inline admin/index.php]: Spaces vs tabs

File: admin/index.php [hunk 2]
```diff
@@ -73,6 +77,7 @@
 						';
 
 						foreach($fields as $key => $field){
+                                                        if ($key == 'platform') { continue; };
```
Comment (andi34) [inline admin/index.php:80]: Same here

File: admin/index.php (hunk 1 above)
Comment (jacques42) [reply admin/index.php]: done

File: admin/index.php (hunk 2 above)
Comment (jacques42) [reply admin/index.php:80]: done

**Comment by jacques42** [comment]
Untabified admin/index.html as a whole - there was plenty of tabs. Also tested the change on Windows along with 2.10.0 (stable2), works perfectly well there.

**Pull request merged**